qiita_pet/templates/study_ajax/processing_study.html

```diff
@@ -0,0 +1,173 @@
+<script type="text/javascript">
+  //the network object itself, + nodes list and edge list
+  var proc_network = null;
+  var nodes = new vis.DataSet([{id: 1, label: "{{artifact_type}}", group: 'type'}]);
+  var edges = new vis.DataSet();
+  //current commands available for selected nodes
+  var commands = null;
+  //currently selected nodes
+  var curr_nodes = [1];
+
+  function validate_node_id(node_id, action){
+      if (node_id.length !== 1) {
+          alert("Can only " + action + " one command at a time");
+          return -1;
+      }
+      // node_id only has one element
+      return +node_id[0];
```

Comment:
That is used for casting to int. IMO, I don't think it is needed here, but I haven't played enough with this code to be 100% sure, so I prefer to leave as it is.